app/assets/stylesheets/layout/layout.scss

```diff
@@ -5,17 +5,26 @@
 html, body {
 	height: 100%;
 	font-size: $root-font-size;
+	font-family: $text-font;
 }
 
-body {
-	font-family: $text-font;
-	@include color-and-secondary(color, $color-black $color-white, true, true);
-	@include color-and-secondary(background-color, $background-colors, true, true);
-	font-size: rem(15px);
+body, body.with-cards .card {
+	color: theme-light("text");
+	background-color: theme-light("background");
+}
+
+body.dark, body.dark.with-cards .card {
+	color: theme-dark("text");
+	background-color: theme-dark("background");
+}
 
-	&.with-cards {
-		@include color-and-secondary(background-color, $background-colors-secondary, $on-body: true);
-		@include color-and-secondary(color, $color-white $color-black, $on-body: true);
+body.with-cards {
+	color: $color-black;
+	background-color: theme-light("background-contrast");
+
+	&.dark {
+		color: $color-white;
+		background-color: theme-dark("background-contrast");
 	}
 }
```

Comment:
glad you broke this up
this one one of those times where nesting it does not make it any easier to read